lib/mount.c

```diff
@@ -209,6 +209,11 @@ static int fuse_mount_opt_proc(void *data, const char *arg, int key,
 
 	case KEY_MTAB_OPT:
 		return fuse_opt_add_opt(&mo->mtab_opts, arg);
+
+	case FUSE_OPT_KEY_OPT:
```

Comment:
Could you please add a short comment here explaining what it is for? Like

/* Third party options like 'x-gvfs-notrash' */

Reply:
Done